bug-bounty-list/bug-bounty-list.json

```diff
@@ -3829,20 +3549,10 @@
     "hall_of_fame": true,
     "safe_harbor": ""
   },
-  {
-    "program_name": "Instagram",
-    "policy_url": "https://www.facebook.com/whitehat/bounty/",
-    "submission_url": "",
-    "launch_date": "",
-    "cash_reward": true,
-    "swag": true,
-    "hall_of_fame": true,
-    "safe_harbor": ""
-  },
   {
     "program_name": "Instructure",
     "policy_url": "https://blog.bugcrowd.com/private-bug-bounty-program-instructure/",
```

Comment:
This seems to 404 – I see there's https://www.bugcrowd.com/customer/instructure/ but that's not really a policy. I'd only keep programs in the list if we can point at a minimum to a basic policy and an email.